core/src/main/java/hudson/model/listeners/ItemListener.java

```diff
@@ -57,6 +58,18 @@ public void onCreated(Item item) {
     }
 
     /**
+     * Called before a job is copied into a new parent, providing the ability to veto the copy operation before it
+     * starts.
+     *
+     * @param src the item being copied
+     * @param parent the proposed parent
+     * @throws Failure to veto the operation.
+     * @since TODO
+     */
+    public void onCheckCopy(Item src, ItemGroup parent) throws Failure {
```

Comment:
Failure knows how to render the HTML response. Would you be ok if I update the jacadoc of Dailure to reflect its current usage